src/components/modals/ClaimCompModal.js

```diff
@@ -0,0 +1,170 @@
+  letter-spacing: 0.7px;
+  color: #0b0412;
+`
+
+const ExplainerText = styled.div`
+  width: 95%;
+  margin: 40px 149px 20px 165px;
+  font-family: "NeueHaasGroteskDisp Pro Md", sans-serif;
+  font-size: 11px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.55px;
+  text-align: center;
+  color: #17111e;
+`
+
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 45%;
+  margin-bottom: 30px;
+  @media ${device.mobile}{
+    width: 90%;
+  }
+`
+
+const ContentBox = styled.div`
+  width: 100%;
+  border-radius: 9.9px;
+  overflow: hidden;
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.22);
+  background-color: white;
+  &>:not(:last-child) {
+    border-bottom: solid 0.8px rgba(0,0,0,0.1);
+  }
+`
+
+class ClaimCompModal extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  claim = (gasEfficient) => {
+    if(gasEfficient){
+      this.props.efficientClaim()
+    } else {
+      this.props.fullClaim()
+    }
+    EventBus.$emit("close-modal")
+  }
+
+  render() {
+    const {avatarCompBalance, compBalance, efficientCompClaimGasEstimate, compClaimGasEstimate} = compoundStore
+    const efficientClaimPrecentage = parseInt(percentage(avatarCompBalance, compBalance))
+    return (
+      <Container>
+        <Header>
+          <Title>Claim Comp</Title>
+        </Header>
+        <Flex justifyAround wrap style={{margin: "-30px 5% 0 5%"}}>
+          <Body>
+            <ContentBox>
+                <Flex style={{padding: "22px"}} justifyBetween>
+                  <Text>COMP</Text>
+                  <Text>{displayNum(avatarCompBalance, 4)} COMP</Text>
+                </Flex>
+                <Flex style={{padding: "22px"}} justifyBetween>
+                  <Text>Gas</Text>
+                  <Text>{displayNum(efficientCompClaimGasEstimate, 4)} ETH</Text>
+                </Flex>
+            </ContentBox>
+            <ExplainerText>
+              Gas efficient claim process that will retrive {efficientClaimPrecentage}% of COMP
+            </ExplainerText>
+            <ModalButton onClick={()=> this.claim(true)}>
+               <span>Efficient Claim</span>
+            </ModalButton>
+          </Body>
+          <Body>
+            <ContentBox>
+                <Flex style={{padding: "22px"}} justifyBetween>
+                  <Text>COMP</Text>
+                  <Text>{displayNum(compBalance, 4)} COMP</Text>
+                </Flex>
+                <Flex style={{padding: "22px"}} justifyBetween>
+                  <Text>Gas</Text>
+                  <Text>{displayNum(compClaimGasEstimate, 4)} ETH</Text>
+                </Flex>
+            </ContentBox>
+            <ExplainerText>
+              Full calim process that will retrive 100% of COMP but with less gas efficiency.
+            </ExplainerText>
+            <ModalButton onClick={() => this.claim()}>
```

Comment:
don't be cheap. put `false` in the input parameter.

Reply:
done!